zhusuan/distributions/univariate.py

```diff
@@ -163,6 +164,140 @@ def _log_prob(self, given):
     def _prob(self, given):
         return tf.exp(self._log_prob(given))
 
+class HalfNormal(Distribution):
+    """
+    The class of univariate HalfNormal distribution.
+    See :class:`~zhusuan.distributions.base.Distribution` for details.
+
+    .. warning::
+
+         The order of arguments `logstd`/`std` will change to `std`/`logstd`
+         in the coming version.
```

Comment:
Just change it and remove this warning (in the doc string and in the code)

Reply:
maybe we need to keep the consistency between Normal and FoldNormal and not change the order since the two distributions are quite similar ? @thjashin